client_server_http2_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2013 The Gorilla WebSocket Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build go1.14
+// +build go1.14
```

Comment:
this was necessary because https://github.com/gorilla/websocket/pull/788/files#diff-647b905ba2feaa4a134811cd904e1fd3af5d28129e9b9031cfa4fc127f5c55fdR20 was not present prior to go version 1.14